### PR TITLE
Feature testing

### PR DESCRIPTION
**Pull request by meyerhoferc**
Updates feature testing for clicking tabs and viewing content on a legislator show.

Updates feature testing for user dashboard and user following/unfollowing a legislator. The general strategy for this was to send an HttpRequest to the `user_show` or `legislator_detail` view with the user as part of the request (so they were sort of 'logged in'). The assertions are for the body of the response that comes back from the view. The `b'` in the strings is for bytecode, since that's the datatype of the HttpResponse body. 

All tests pass.

### CODE REVIEW

File: deploy_tools/fabfile.py [hunk 1]
```diff
@@ -64,11 +66,15 @@ def _update_secrets(source_folder):
...
     run(f'cd {source_folder} && ../virtualenv/bin/python manage.py collectstatic --noinput')
 
 def _update_database(source_folder):
     run(f'cd {source_folder} && ../virtualenv/bin/python manage.py migrate --noinput')
 
 def _update_images(source_folder):
-    run(f'cd {source_folder} && mkdir public_officials/static/public_officials/images/profiles')
+    images_folder = source_folder + '/pubic_officials/static/public_officials/images/profiles'
+    if not exists(images_folder):
+        run(f'cd {source_folder} && mkdir public_officials/static/public_officials/images/profiles')
```
Comment (cmacaulay) [inline deploy_tools/fabfile.py]: @meyerhoferc thank you for being so on top of what has to be done for deployment!

File: public_officials/functional_tests/test_public_officials.py [hunk 2]
```diff
@@ -67,6 +67,103 @@ def test_checks_for_content_on_official_show(self):
...
+                                                   twitter_id="RepDianaDeGette")
+
+            self.browser.get(self.live_server_url + '/legislators/%d/' % legislator.id)
+            self.browser.find_element_by_css_selector('#vote-tab').click()
+            voting_history_text = self.browser.find_element_by_css_selector('.voting-history').text
+            self.assertIn('Voting History', voting_history_text)
+            self.assertIn('Proposition', voting_history_text)
+            self.assertIn('Vote', voting_history_text)
+            self.assertIn('Result', voting_history_text)
+
```
Comment (cmacaulay) [inline public_officials/functional_tests/test_public_officials.py]: 🎉 🎉 amazing work!

File: public_officials/functional_tests/test_user_dashboard.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            self.assertNotIn(b'House', user_homepage_response.content)
+            self.assertNotIn(b'CO-D', user_homepage_response.content)
+            self.assertNotIn(b'Display Tweets from Diana', user_homepage_response.content)
+            self.assertNotIn(b'Unfollow Diana', user_homepage_response.content)
+
+            legislator_show_request = self.factory.get(f'/legislators/{legislator.id}')
+            legislator_show_request.user = user
+            legislator_show_response = legislator_detail(legislator_show_request, legislator.id)
+            self.assertNotIn(b'Unfollow', legislator_show_response.content)
+            self.assertIn(b'Follow', legislator_show_response.content)
```
Comment (cmacaulay) [inline public_officials/functional_tests/test_user_dashboard.py]: 👏 👏 testing!

File: public_officials/templates/public-officials/detail.html [hunk 4]
```diff
@@ -55,12 +55,12 @@
...
-        <li><a href="#tab-3">Industry Contributions</a></li>
-        <li><a href="#tab-4">Sponsored Bills</a></li>
-        <li><a href="#tab-5">Recent Decisions</a></li>
-        <li><a href="#tab-6">Data Dashboard</a></li>
+        <li id='about-tab'><a href="#tab-1">About</a></li>
+        <li id='org-tab'><a href="#tab-2">Organization Contributions</a></li>
+        <li id='ind-tab'><a href="#tab-3">Industry Contributions</a></li>
+        <li id='bill-tab'><a href="#tab-4">Sponsored Bills</a></li>
+        <li id='vote-tab'><a href="#tab-5">Recent Decisions</a></li>
+        <li id='data-tab'><a href="#tab-6">Data Dashboard</a></li>
```
Comment (cmacaulay) [inline public_officials/templates/public-officials/detail.html]: I like this a lot better.

File: public_officials/functional_tests/test_public_officials.py [hunk 5]
```diff
@@ -67,6 +67,103 @@ def test_checks_for_content_on_official_show(self):
...
+                                                   twitter_id="RepDianaDeGette")
+
+            self.browser.get(self.live_server_url + '/legislators/%d/' % legislator.id)
+            self.browser.find_element_by_css_selector('#org-tab').click()
+            organization_contributions_text = self.browser.find_element_by_css_selector('.organization-contributors').text
+            self.assertIn('Contributions by Organization', organization_contributions_text)
+            self.assertIn('Organization', organization_contributions_text)
+            self.assertIn('Total', organization_contributions_text)
+            self.assertIn('Individual Donations', organization_contributions_text)
+            self.assertIn('PACs', organization_contributions_text)
```
Comment (TheKaterTot) [inline public_officials/functional_tests/test_public_officials.py]: Beautiful new tests!

File: public_officials/functional_tests/test_user_dashboard.py [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+            user.legislator_set.remove(legislator)
+            user_homepage_request = self.factory.get('/acounts/profile/')
+            user_homepage_request.user = user
+            user_homepage_response = user_show(user_homepage_request)
+            self.assertNotIn(b'Diana DeGette', user_homepage_response.content)
+            self.assertNotIn(b'House', user_homepage_response.content)
+            self.assertNotIn(b'CO-D', user_homepage_response.content)
+            self.assertNotIn(b'Display Tweets from Diana', user_homepage_response.content)
+            self.assertNotIn(b'Unfollow Diana', user_homepage_response.content)
```
Comment (TheKaterTot) [inline public_officials/functional_tests/test_user_dashboard.py]: 💯

File: public_officials/models.py [hunk 7]
```diff
@@ -1,6 +1,5 @@
 from django.db import models
 from django.contrib.auth.models import User
-import pdb
```
Comment (TheKaterTot) [inline public_officials/models.py]: Good cleanup!

**Pull request merged**